src/legends/continuous-color-legend.js

```diff
@@ -59,24 +60,26 @@ function ContinuousColorLegend({startColor, midColor, endColor, startTitle,
   }
   colors.push(endColor);
   return (
-    <div className={`rv-continuous-color-legend ${className}`} style={{width, height}}>
-      <div
+    <LegendContainer>
+      <div className={`rv-continuous-color-legend ${className}`} style={{width, height}}>
+        <div
         className="rv-gradient"
         style={{background: `linear-gradient(to right, ${colors.join(',')})`}} />
-      <div className="rv-legend-titles">
-        <span className="rv-legend-titles__left">
-          {startTitle}
-        </span>
-        <span className="rv-legend-titles__right">
-          {endTitle}
-        </span>
-        {midTitle ?
-          <span className="rv-legend-titles__center">
-            {midTitle}
-          </span> :
+        <div className="rv-legend-titles">
+          <span className="rv-legend-titles__left">
+            {startTitle}
+          </span>
+          <span className="rv-legend-titles__right">
+            {endTitle}
+          </span>
+          {midTitle ?
```

Comment:
while were your in here, can this change to be:

```jsx
{Boolean(midTitle) && <span className="rv-legend-titles__center">{midTitle}</span>}
```
Seeing ternarys with null branches looks bad